### PR TITLE
Fix some typos in the french translation

### PR DESCRIPTION
**Pull request by smalesys**

### CODE REVIEW

File: content/fr/config.md [hunk 1]
```diff
@@ -1,22 +1,22 @@
 ## III. Configuration
 ### Stockez la configuration dans l'environnement
 
-La *configuration* d'une application est tout ce qui est susceptible de varier entre des [déploiements](./codebase) (validation, production, environnement de développement, etc). Cela inclut :
+La *configuration* d'une application est tout ce qui est susceptible de varier entre des [déploiements](./codebase) (validation, production, environnement de développement, etc.). Cela inclut :
```
Comment (dmathieu) [inline content/fr/config.md]: not sure we need the dot here.

Comment (smalesys) [reply content/fr/config.md]: Ya it's not strictly required. Usually abbreviations are ended with a dot, some correctors consider this as a mistake if there is none. 
But sure, we can skip it if you prefer, it's absolutely not mandatory. Just an expression of my nevrotic mind :)

Comment (dmathieu) [reply content/fr/config.md]: Oh, due to it being an abbreviation, of course.
I thought you were adding it as the end of a phrase, which seemed weird for a parenthesis.

**Comment by raul** [comment]
Thank you both! Looks like there's a conflict caused by https://github.com/heroku/12factor/pull/152/commits/58e0bb9f7bafaf55c5b76f45a8bbc09edc4d1c1c, which added systemd - would you mind taking a look at it @smalesys ?

**Comment by raul** [comment]
The specific change in the French content is [here](https://github.com/heroku/12factor/pull/152/commits/58e0bb9f7bafaf55c5b76f45a8bbc09edc4d1c1c#diff-728018e82d3d70fadb3a509f77a5ae70).

**Comment by smalesys** [comment]
Done :)

**Comment by raul** [comment]
Wonderful, thanks a lot! :)

**Pull request merged**